### ISSUE TITLE
Rectangle Select and Ellipse Select not working on Chrome 120 on Fedora Linux

### ISSUE THREAD

**Issue by clemensjur** [photopea/photopea#6238]
OS: Fedora 39 Workstation
Browser: Google Chrome Version 120.0.6099.109 (Official Build) (64-bit)
The problem happens in the Incognito Mode, too: Yes

**This must be done to see the problem**
1. Go to www.photopea.com
2. Open an Image File (tested Formats: JPG, PNG, WEBP)
3. Try to select an area using the Rectangle Select Tool or Ellipse Select Tool
4. See that instead of selecting the logged action is "Deselect"

The problem started happening after updating to Google Chrome 120 and doesn't happen on Firefox.

**Comment by photopea** [comment]
That is really strange. Could you try to disable all your browser extensions? Do you see any errors in the web console?

**Comment by clemensjur** [comment]
Disabling all extensions doesn't change anything as far as I could tell. And the errors in the console all appeare to be ad service related. 

Here is a short clip of this behavior.
[photopea_issue_6238.webm](https://github.com/photopea/photopea/assets/16306561/4ff82a82-1f7b-404b-847f-dee5361d5137)

**Comment by photopea** [comment]
It is very strange. Do you have any software which emulates mouse behaviour? It is like you released the mouse button right after clicking. Or does your mouse do double-clicks on its own?

**Comment by clemensjur** [comment]
I was able to look into this more now and found that the issue dosen't stem from photopea. 

This problem appeared because I was using photopea in a VMWare virtual machine on a different system than usual that had the Setting "Optimize mouse for games" (Edit > Preferences > Input > Optimize mouse for games) set to "Never" which (I assume) changes the way the VM captures mouse input and subsequently messes up photopeas ability to correctly process this information.

If anyone ever encounters the same issue set the aforementioned option to either "Automatic" or "Always". 
With the option set to "Automatic" I experienced the issue as well, however only sometimes and a browser restart often fixed it. 
With the option set to "Always" I haven't experienced the issue again.

@photopea thanks for your time and help! I'm closing this issue now.

Cheers!
